tbx/core/blocks.py

```diff
@@ -527,6 +533,7 @@ def clean(self, value):
     class Meta:
         icon = "image"
         template = "patterns/molecules/streamfield/blocks/photo_collage_block.html"
+        value_class = ButtonLinkStructValue
```

Comment:
**Comment**: As with `EventLinkStructValue`, this feels a little bit strangely convenient, or magical. The mismatch between `PhotoCollageBlock` and `ButtonLinkStructValue` sticks out like a sore thumb. What would happen if another subblock needed to add bespoke behaviour, and then we need to combine them?

I think this is one to bear in mind for the future, rather than address now.